packages/website/docs/xy-charts/Area.mdx

```diff
@@ -58,9 +58,11 @@ accessor. A common configuration is to utilize the data's index:
 <XYWrapper {...areaProps()} showContext="full" y={[(d) => d.y, (d) => d.y1, (d) => d.y2]} color={(d, i) => ['red', 'green', 'blue'][i]}/>
 
 ## Dealing with small values
-If your data has small or zero values leading to some parts of the area to become invisible, you can force those area segments to have 1px height
-despite their actual value by setting `minHeight1Px` to `true`. This can be useful if you want to visually emphasize that the data behind the chart
-is defined but just very small.
+If your data has small or zero values leading to some parts of the area to become invisible, you can use the `minHeight` property to set a minimum height for the area. This can be useful if you want to visually emphasize that the data behind the chart is defined but just very small.
```

Comment:
Do we want to add a note there indicating we want to deprecate `minHeight1px`? Otherwise right now there's no mention on the website, just 2 examples right next to each other

Reply:
@lee00678 I was thinking about it but decided not to (forgot to remove the example). The property will be marked as deprecated in the code asking users to use `minHeight`

Reply:
@lee00678 I was thinking about it but decided not to (forgot to remove the example). The property will be marked as deprecated in the code asking users to use `minHeight`

Reply:
@rokotyan I will remove the `minHeight1px` example then

Reply:
@lee00678 Thanks